src/components/common/Builder/FormBuilder/types.ts

```diff
@@ -0,0 +1,35 @@
+import type { FC } from "react";
+import type {
+    SubmitHandler,
+    UseFormStateReturn,
+    ControllerFieldState,
+    ControllerRenderProps,
+} from "react-hook-form";
+import type { PropertyMeta } from "../../../../services/hubspot/types";
```

Comment:
Use `alias` where possible

Reply:
We've already agreed to do it later. I'll take care of techdebt in the [main story](https://app.shortcut.com/deskpro/story/122828/hubspot-field-mappings)